frontend/src/routes/(app)/report/[owner]/[report]/+page.svelte

```diff
@@ -94,27 +83,31 @@
 	}
 </script>
 
+{#if showConfirmDelete !== -1}
+	<Confirm
+		message="Are you sure you want to delete this element?"
+		on:cancel={() => {
+			showConfirmDelete = -1;
+		}}
+		on:confirm={() => {
+			deleteElement(showConfirmDelete);
+			showConfirmDelete = -1;
+		}}
+	/>
+{/if}
 <div class="w-full h-full bg-yellowish overflow-scroll">
 	<div
 		class="flex flex-col max-w-4xl m-auto bg-background px-10 pb-20 md:mt-6 md:mb-6 sm:mt-0 sm:mb-0 rounded shadow"
 	>
 		<div class="flex items-center mt-12 justify-between">
-			<h1
-				class="text-5xl mr-6 text-grey-darkest"
-				contenteditable={isEdit ? true : false}
-				on:blur={(e) => updateReportName(e)}
-			>
+			<h1 class="text-5xl mr-6 text-grey-darkest">
```

Comment:
How can one update the name of a report now?

Reply:
In the settings!